### PR TITLE
MDS-1153 style and content for public home page

### PR DESCRIPTION
**Pull request by JustinMacaulay**
Added some base styling for the new public site as well as the landing page, header and footer.

### CODE REVIEW

File: frontend-public/src/components/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+        <br />
+        <p>
+          MineSpace is a pilot project as a part of the MDS digital product. MineSpace will be the
+          portal through which mines can interact with the Ministry. In time, Mines will be able to
+          submit documentation to the Ministry, as well as check the status of reporting
+          requirements, and documentation that they have submitted to the ministry for review. The
+          pilot project will run in the spring of 2019 with full integration in early 2020.
+        </p>
+      </Col>
+      <Col xs={0} md={2} lg={6} />
```
Comment (mmumby) [inline frontend-public/src/components/LandingPage.js]: What is the purpose of these empty columns?

Comment (JustinMacaulay) [reply frontend-public/src/components/LandingPage.js]: I need them to be there in lg and md to add white space around the content but not on the xs screen size.

File: frontend-public/src/styles/settings/theme.scss [hunk 2]
```diff
@@ -0,0 +1,8 @@
+// theme.scss holds overrides for ant-design
+// refer to (https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less) for ant design variable names;
+$blue: #003366;
+$white: #f1f1f2;
+$violet: #5e46a1;
+
```
Comment (mmumby) [inline frontend-public/src/styles/settings/theme.scss]: Violet is not needed as its an MDS color

File: frontend-public/src/styles/settings/theme.scss [hunk 3]
```diff
@@ -0,0 +1,8 @@
+// theme.scss holds overrides for ant-design
+// refer to (https://github.com/ant-design/ant-design/blob/master/components/style/themes/default.less) for ant design variable names;
+$blue: #003366;
+$white: #f1f1f2;
+$violet: #5e46a1;
```
Comment (mmumby) [inline frontend-public/src/styles/settings/theme.scss]: I think all of these can be removed actually, they aren't being used anywhere. This file can be left blank unless you're overriding anything with those variables.

File: frontend-public/src/components/LandingPage.js (hunk 1 above)
Comment (mmumby) [reply frontend-public/src/components/LandingPage.js]: Okay, 

There is a better way to accomplish this than using empty elements, especially in this quantity. 
I can assist in this area if you'd like.

File: frontend-public/src/styles/settings/theme.scss (hunk 3 above)
Comment (JustinMacaulay) [reply frontend-public/src/styles/settings/theme.scss]: made the file empty, I needed it for the web pack loader but forgot about the variables in it. Thanks.

**Pull request merged**